scripts/start.py

```diff
@@ -178,6 +184,14 @@ def main(args: Optional[Sequence[str]] = None) -> None:
             skip_sdk_update_check=True,
             port=PORT_NUMBER_FOR_GAE_SERVER))
 
+        if parsed_args.contributor_dashboard_debug:
+            debug_requests = (
+                contributor_dashboard_debug.ContributorDashboardDebugInitializer
+                (
+                base_url='http://localhost:%s' % PORT_NUMBER_FOR_GAE_SERVER
```

Comment:
this indentation is hard to read. Maybe shorten `ContributorDashboardDebugInitializer`, e.g. to `Initializer` so that we have `contributor_dashboard_debug.Initializer`?